yt/data_objects/static_output.py

```diff
@@ -216,6 +217,70 @@ def __get__(self, instance, owner):
     def __set__(self, instance, value):
         self.data[instance] = value
 
+class ConstantContainer(object):
```

Comment:
Can these go into a separate file?  And I think for my own understanding, just a comment about what each is supposed to do (doesn't have to be docstrings or anything extensive) would be helpful.